icu4j/main/tests/core/src/com/ibm/icu/dev/test/format/MeasureUnitTest.java

```diff
@@ -3103,12 +3103,12 @@ static void generateCXXConstants() {
                     throw new IllegalStateException();
                 }
                 checkForDup(seen, name, unit);
-                System.out.printf("MeasureUnit *MeasureUnit::create%s(UErrorCode &status) {\n", name);
-                System.out.printf("    return MeasureUnit::create(%d, %d, status);\n",
+                System.out.printf("MeasureUnit *MeasureUnit.create%s(UErrorCode &status) {\n", name);
+                System.out.printf("    return MeasureUnit.create(%d, %d, status);\n",
```

Comment:
This is breaking the C++ output again.

Reply:
Please revert changes to MeasureUnitTest.java

Reply:
Good catch, this file needs to be reverted

Reply:
Done

Reply:
Thanks! (Awaiting push of the commit.)